cfdilib/templates/cfdv33.xml

```diff
@@ -78,6 +78,12 @@
                     {% endif %}
                 </cfdi:Impuestos>
             {% endif %}
+            {% if line.customs_number %}
```

Comment:
I get the following error even with the if and sending customs_number as False. @nhomar @luistorresm  
`
Element '{http://www.sat.gob.mx/cfd/3}InformacionAduanera', attribute 'NumeroPedimento': [facet 'length'] The value '' has a length of '0'; this differs from the allowed length of '21'., line 65
`